subsys/net/ip/net_if.c

```diff
@@ -481,6 +481,13 @@ enum net_verdict net_if_send_data(struct net_if *iface, struct net_pkt *pkt)
 	}
```

Comment:
LGTM, could you change the commit subject to `net: iface: ....` as the first part is not meant to be a file/directory but a logical component that the commit is changing.

Reply:
done.